doc/web-doc/doc-loader.js

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs-extra'
+import { compareSync } from 'dir-compare'
+
+const src = '../adr/'
+const dest = 'doc-sources/adr-source/'
+
+const errorCallback = err => {
+  if (err) throw err;
+}
+
+const options = {
+  excludeFilter: '.DS_Store',
+  compareSize: true
+}
+
+// Compare folders
+
+const comparison = compareSync(src, dest, options)
+
+// Copy missing files and folders
+
+if (!comparison.same) {
+  comparison.diffSet.forEach(dif => {
+    if (dif.state === 'left' && dif.path1 === src) {
+      fs.copySync(src + dif.name1, dest + dif.name1, { recursive: true }, errorCallback)
+    }
+  })
+}
```

Comment:
Running the hereby lint/format `yarn` commands lends updates for this file for me locally.

Reply:
Fixed.